app/scripts/constants.js

```diff
@@ -12,6 +12,7 @@ window.OPENSHIFT_CONSTANTS = {
     "cli":                     "https://docs.openshift.org/latest/cli_reference/overview.html",
     "get_started_cli":         "https://docs.openshift.org/latest/cli_reference/get_started_cli.html",
     "basic_cli_operations":    "https://docs.openshift.org/latest/cli_reference/basic_cli_operations.html",
+    "build-triggers":          "https://docs.openshift.org/latest/dev_guide/builds.html#build-triggers",
```

Comment:
@jwforres @sg00dwin FYI, we'll need to update the enterprise-3.3 branch as well for the new doc link